docs/releasenotes/index.rst

```diff
@@ -10,14 +10,16 @@ You can learn more about where we might be going with future releases by:
 * Stopping by the ``#clusterhq`` channel on ``irc.freenode.net``.
 * Visiting our GitHub repository at https://github.com/ClusterHQ/flocker.
 
-Next Release
-============
+v1.8.0
+======
 
 * The Docker plugin is now able to use datasets created directly via Flocker so long as the metadata has a matching ``"name"`` value.
 * Better error reporting for the Docker plugin.
 * Added a new REST API for looking up node identity by era; eras are reset after every reboot.
   This allows interacting with Flocker in a robust way across reboots without getting stale data.
```

Comment:
Small change (optional - I think this just reads a bit easier):
From: This allows interacting with Flocker in a robust way across reboots without getting stale data.
To: This allows robust interaction with Flocker across reboots without getting stale data.

Is it worth specifying what the new REST API is called?